microbootstrap/bootstrappers/litestar.py

```diff
@@ -51,24 +52,28 @@ def bootstrap(self) -> None:
 @LitestarBootstrapper.use_instrument()
 class LitestarSwaggerInstrument(SwaggerInstrument):
     def bootstrap_before(self) -> dict[str, typing.Any]:
-        class LitestarOpenAPIController(openapi.OpenAPIController):
-            path = self.instrument_config.swagger_path
-            if self.instrument_config.swagger_offline_docs:
-                swagger_ui_standalone_preset_js_url = (
+        swagger_render_plugin: typing.Final = (
+            SwaggerRenderPlugin(
+                js_url=f"{self.instrument_config.service_static_path}/swagger-ui-bundle.js",
+                css_url=f"{self.instrument_config.service_static_path}/swagger-ui.css",
+                standalone_preset_js_url=(
                     f"{self.instrument_config.service_static_path}/swagger-ui-standalone-preset.js"
-                )
-                swagger_bundle_path: str = f"{self.instrument_config.service_static_path}/swagger-ui-bundle.js"
-                swagger_css_url: str = f"{self.instrument_config.service_static_path}/swagger-ui.css"
+                ),
+            )
+            if self.instrument_config.swagger_offline_docs
+            else SwaggerRenderPlugin()
```

Comment:
Or you're telling that we should not select render plugin if offline docs flag is set to `False`?